specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-03-30-preview/examples/listConfigurationProfileAssignmentIntentsByResourceGroup.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "resourceGroupName": "myResourceGroupName",
+    "vmName": "myVMName",
+    "api-version": "2021-03-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": [
+        {
```

Comment:
response should be:
{
  "value": []
}

Reply:
Thank you! Resolved!